bedrock/settings/base.py

```diff
@@ -247,7 +246,8 @@ def lazy_langs():
     'fxa_form',
     'navigation',
     'newsletter_form',
-    'ui'
+    'send_to_device',
```

Comment:
Oooooooooohhhhhhhh I remember. This has to be a default file because if you set an `ftl_file` on the view then it will switch to using the Fluent system to determine page activation. Got it 👍

Reply:
Yeah I initially tried to add it only to the views that needed it, but figured this would be an issue. Once we migrate all the pages that rely on it to Fluent, we could always remove this file from the default list.

Fwiw, I think this is the last file we might consider to be global (for now). I'll move onto migrating the firefox download pages next 🎉